content/en/tutorials/lotus/grafana-dashboard.md

```diff
@@ -0,0 +1,161 @@
+
+## Prerequisites
+
+- You have a Linux or Mac based system.
+- You have root access to install software
+- You have lotus node already running
+
+**Note:** These instructions have been tested on Ubuntu 23.04 and on Mac M1.
+
+## Install and start Prometheus
+
+### On Ubuntu:
+
+```
+# install prometheus
+sudo apt-get install prometheus
+
+# copy the prometheus.yml config to the correct directory
+sudo cp metrics/prometheus.yml /etc/prometheus/prometheus.yml
+
+# start prometheus
+sudo systemctl start prometheus
+
+# enable prometheus on boot (optional)
+sudo systemctl enable prometheus
+```
+
+### On Mac:
+
+```
+# install prometheus
+brew install prometheus
+
+# start prometheus
+prometheus --config.file=lotus/metrics/prometheus.yml
+```
+
+## Install and start Grafana
+
+### On Ubuntu:
+
+```
+# download the Grafana GPG key in our keyring
+wget -q -O - https://packages.grafana.com/gpg.key | gpg --dearmor | sudo tee /usr/share/keyrings/grafana.gpg > /dev/null
+
+# add the Grafana repository to our APT sources
+echo "deb [signed-by=/usr/share/keyrings/grafana.gpg] https://packages.grafana.com/oss/deb stable main" | sudo tee -a /etc/apt/sources.list.d/grafana.list
+
+# update our APT cache
+sudo apt-get update
+
+# now we can install grafana
+sudo apt-get install grafana
+
+# start grafana
+sudo systemctl start grafana-server
+
+# start grafana on boot (optional)
+sudo systemctl enable grafana-server
+```
+
+### On Mac:
+
+```
+brew install grafana
+brew services start grafana
+```
+
+You should now have Prometheus and Grafana running on your machine where Prometheus is already collecting metrics from your Lotus node (if its running) and saving it to a database.
+
+You can confirm everything is setup correctly by visiting:
+- Prometheus (http://localhost:9090): You can open the metric explorer and view any of the aggregated metrics scraped from Lotus
+- Grafana (http://localhost:3000): Default username/password is admin/admin, remember to change it after login.
+
+## Add Prometheus as datasource in Grafana
+
+1. Log in to Grafana using the web interface.
+2. Navigate to "Home" > "Connections" > "Data Sources."
+3. Click "Add data source."
+4. Choose "Prometheus."
+5. In the "HTTP" section, set the URL to http://localhost:9090.
+6. Click "Save & Test" to verify the connection.
+
+## Import one of the existing dashboards in lotus/metrics/grafana
+
+1. Log in to Grafana using the web interface.
+2. Navigate to "Home" > "Dashboards" > Click the drop down menu in the "New" button and select "Import"
+3. Paste any of the existing dashboards in lotus/metrics/grafana into the "Import via panel json" panel.
+4. Click "Load"
+5. Select the Prometheus datasource you created earlier
+6. Click "Import"
+
+### Available Dashboards
```

Comment:
Point to https://github.com/filecoin-project/lotus/blob/master/metrics/README.md#available-downloads from https://github.com/filecoin-project/lotus/pull/12934 ?